static/app/views/performance/browser/interactionTable.tsx

```diff
@@ -65,4 +69,13 @@ function InteractionsTable() {
   );
 }
 
+const getFriendlyActionName = (action: string) => {
```

Comment:
l: don't think it makes a difference if the action name is friendly :p

```suggestion
const getSpanAction = (action: string) => {
```

Reply:
haha, it converts the transaction.op like 'ui.action.click', to something more friendly like 'click' which is a bit friendlier 🤷 .